Code/Framework/AzCore/AzCore/Serialization/EditContext.h

```diff
@@ -514,6 +525,54 @@ namespace AZ
         return this;
     }
 
+    //=========================================================================
+    // ClassElement
+    //=========================================================================
+    template<class T>
+    inline EditContext::ClassBuilder* EditContext::ClassBuilder::ClassElement(Crc32 elementIdCrc, const char* description, T memberVariable)
+    {
+        if (IsValid())
+        {
+            using ElementTypeInfo = typename SerializeInternal::ElementInfo<T>;
+            AZ_Assert(
+                m_classData->m_typeId == AzTypeInfo<typename ElementTypeInfo::ClassType>::Uuid(),
+                "Data element (%s) belongs to a different class!", description);
+
+            // Not really portable but works for the supported compilers
+            size_t offset =
+                reinterpret_cast<size_t>(&(reinterpret_cast<typename ElementTypeInfo::ClassType const volatile*>(0)->*memberVariable));
+            // offset = or pass it to the function with offsetof(typename ElementTypeInfo::ClassType,memberVariable);
```

Comment:
question: I'm not sure I understand this comment, what is meant by `offset =`?

Reply:
See above.